event-log/src/test/scala/io/renku/eventlog/init/EventPayloadTableCreatorSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.eventlog.init
+
+import cats.effect.IO
+import ch.datascience.interpreters.TestLogger
+import ch.datascience.interpreters.TestLogger.Level.Info
+import doobie.implicits._
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpec
+
+class EventPayloadTableCreatorSpec extends AnyWordSpec with DbInitSpec with should.Matchers {
+
+  protected override lazy val migrationsToRun: List[Migration] = List(eventLogTableCreator)
+
+  "run" should {
+    "fail if the 'event' table does not exist" in new TestCase {
+
+      tableExists("event")         shouldBe false
```

Comment:
If we'd run all the previous migrations, we'd probably need to use `dropTable('event')` here.